tsconfig.json

```diff
@@ -22,7 +22,8 @@
     "target": "ES2020",
     "resolveJsonModule": true,
     "typeRoots": ["./node_modules/@types"],
-    "types": ["node", "mocha"]
+    "types": ["node", "mocha"],
+    "lib": [ "ES2016" ]
```

Comment:
What was the issue without this config?

Reply:
Using no lib option in compiler options, it will default to `[ "DOM", "ES2016" ]`, and we don't want to have the "DOM" library included because NodeJS has no support for this.